meta-mender-demo/recipes-extended/images/mender-demo-full-delta.bb

```diff
@@ -0,0 +1,9 @@
+require recipes-extended/images/mender-demo-full-rw.bb
+
+IMAGE_FEATURES_append = " read-only-rootfs"
+IMAGE_INSTALL_append = " mender-binary-delta"
+
+# See https://tracker.mender.io/browse/MEN-3513 and
+# https://tracker.mender.io/browse/MEN-3781 and
+# https://tracker.mender.io/browse/MEN-3912.
+EXTRA_IMAGECMD_ext4_append = "${@bb.utils.contains('IMAGE_FEATURES', 'read-only-rootfs', ' -O ^64bit -O ^has_journal', '', d)}"
```

Comment:
This shouldn't be necessary, since this fix [already exists in the meta-mender-commercial layer](https://github.com/mendersoftware/meta-mender/blob/master/meta-mender-commercial/conf/layer.conf), which you are forcing to include because of `MENDER_INSTALL_append = " mender-binary-delta"`.

Reply:
Thanks, would that hold true for all images then?
([This one](https://github.com/mendersoftware/meta-mender/blob/master/meta-mender-demo/recipes-extended/images/mender-image-full-cmdline-rofs.bb) was already in the demo layer.)

Reply:
No, not all images. The one you pointed to is there because it is not normally built with the meta-mender-commercial layer. The reason is a bit complicated, but it comes down to security: We have to build an image without commercial components, and then add them in later. This is only for testing, so it doesn't affect the image you are introducing, which should always build with meta-mender-commercial enabled.